scripts/transpile/plugins/babel-plugin-sass-to-es.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+
+// This plugin transpiles sass imports to imports from dist/es
+// this solves the problem for ssr where we have classname mismatch.
+
+function transpileSASS(api, state) {
```

Comment:
```suggestion
function transpileSASS(path, state) {
```

i'd use `path` here (and also lines 40, 41), cause that's what babel is passing here, it's the api of path object